src/utils/fluid_sys.h

```diff
@@ -328,6 +328,29 @@ typedef _private fluid_private_t;
 
 /* Atomic operations */
 
+#if HAVE_PTHREAD_H
+
+#include <stdbool.h>
+
+int pthread_atomic_int_get(const volatile int *atomic);
```

Comment:
I know that currently EE toolchain doesn't support `atomic` library, however, we still have the headers, can't we just keep the names and implement those methods with the same implementation you did with `pthread`?
It will avoid you the change in this header